schemas/json/README.md

```diff
@@ -58,24 +58,35 @@ We explicitly forbid empty JSON arrays to avoid confusion about attributes which
+In UML, it is the convention to name associations and aggregations in singular form. The cardinality is to be taking into account to decide on whether there are none, a single or several elements in the corresponding association or aggregation. 
+In JSON it is the array that declares the set of elements, therefore we choose the plural name for it. The singular name is used for its descriminator (see section on decriminators).
+Typically the plural name is derived by just adding an "s" to the name.
 For example, ``submodelElements`` instead of ``submodelElement`` in case of [Submodel] class.
 
+If plural form made no sense for a property, we kept it as-is (*e.g.*, `isCaseOf`).
+The full list of exceptions is available [as code in aas-core-meta].
+
 If plural form made no sense for an attribute, we kept it as-is (*e.g.*, `isCaseOf`).
 The full list of exceptions is available [as code in aas-core-meta].
 
 [as code in aas-core-meta]: https://github.com/aas-core-works/aas-core-meta/blob/95055a55a8c8f60d75fb48c26eb932ff99945dd2/tests/test_v3rc2.py#L1122
 
-### Primitive attribute value 🠒 JSON string
+### Primitive attribute values
 
-We strictly use only [JSON strings] to represent the attribute values.
+The UML specification uses xsd types. For the mapping of xsd types to JSON types please refer to [Part 2 of the series of the Asset Adminsistration Shell].
+
+There are the following exceptions: For [Property]/value and [Range]/min and [Range]/min the type xs:string is used because the type is only known at run-time. Its type is declared in [Property]/valueType or [Range]/valueType, resp.
+Note: in valueOnly Format of [Part 2 of the series of the Asset Adminsistration Shell] value has the type as declared in Property/valueType.
+
+[Part 2 of the series of the Asset Adminsistration Shell]: https://industrialdigitaltwin.org/wp-content/uploads/2021/11/Details_of_the_Asset_Administration_Shell_Part_2_V1.pdf?__blob=publicationFile&v=10#page=81
 
-[JSON strings]: https://json-schema.org/understanding-json-schema/reference/string.html
 
-This might come as a surprise, given that classes of the meta-model such as [Property] and [Range] allow for boolean and numeric values.
-The reason why we could not map those values to [JSON number] or [JSON boolean] is that the attribute values of the meta-model are based on [XSD types], and not on JSON (see Section [5.7.12 Primitive and Simple Data Types] of the meta-model).
-As [XSD types] do not map to JSON types, we can only represent them as strings.
-We will look more into this problem in the next section.
+### Be aware of possible type mismatches between xsd and JSON types
+
+Type mappings are not trivial. Some sort of type mapping is always needed when using different programming languages and different serialization formats. One way to avoid some JSON type conversion problems would have been to just use [JSON strings] for all primitive attributes values. 
```

Comment:
You mention above that xs:string is always used -- a contradiction?

Reply:
I did not mention this, if it is a copy/paste error